ports/butterflysoup/port.json

```
@@ -0,0 +1,36 @@
+{
+  "version": 3,
+  "name": "butterflysoup.zip",
+  "items": [
+    "Butterfly Soup.sh",
+    "butterflysoup"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Butterfly Soup",
+    "porter": [
+      "all_yall"
+    ],
+    "desc": "A visual novel about gay asian girls playing baseball and falling in love.",
+    "desc_md": null,
+    "inst": "Get the game from itch.io here; https://brianna-lei.itch.io/butterfly-soup (Please consider buying!). Put the contents of the `game` directory into the gamefiles directory in the port.",
+    "inst_md": "Get the game from [Brianna Lei's itch.io](https://brianna-lei.itch.io/butterfly-soup). Place the contents of the `game/` directory into the `port/butterflysoup/gamefiles/` directory.",
+    "genres": [
+      "visual novel"
+    ],
+    "image": {
+      "screenshot": "screenshot.png",
+      "covers": [],
+      "thumbnail": null,
+      "video": null
+    },
+    "rtr": true,
+    "exp": false,
+    "runtime": "renpy_8.1.3.squashfs",
+    "reqs": [],
+    "arch": [
+      "aarch64"
```

Runetimes don't use architectures in combination.

```suggestion
      "aarch64"
```

---

In that case the online editor needs to be updated because its giving me this style as well. Its also what was in the Milk Inside port. In place of the array should I set it to null? or remove the field entirely?

---

(I still have this issue in  new review, though fixed everything else)